controllers/awscluster_controller.go

```diff
@@ -171,6 +171,7 @@ func (r *AWSClusterReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 			patch.WithOwnedConditions{Conditions: []clusterv1.ConditionType{
 				infrav1.PrincipalCredentialRetrievedCondition,
 				infrav1.PrincipalUsageAllowedCondition,
+				infrav1.LoadBalancerReadyCondition,
```

Comment:
Is this change really needed?
Does setting `LoadBalancerReadyCondition` as done in loadbalancer.go would not suffice?

Reply:
Our observation was that it didn't work without having the condition in that list. My understanding is that the patch helper will ignore conditions not owned by the controller. At least that's the impression I'm left with by the documentation [here](https://github.com/kubernetes-sigs/cluster-api/blob/333341b1f72e9ac2fdb40b02b0f114054c68cd5b/util/patch/options.go#L60-L64).

I'll double check, since it's been a while since I submitted this change.

Reply:
cc @richardcase if you have any opinions